Gopkg.toml

```diff
@@ -75,7 +75,7 @@ required = [
 
 [[override]]
   name = "k8s.io/code-generator"
-  version = "kubernetes-1.15.3"
+  version = "kubernetes-1.16.4"
```

Comment:
The version of code generator needs to be 1.16 in order to pick up the change https://github.com/kubernetes/kubernetes/pull/80440

Reply:
We don't bump this in isolation.  We should bump this for all of our K8s dependencies in all of our repos simultaneously, and that should be coupled with a move to 1.15 minimum version.

cc @dgerd

Reply:
In general, we may not want to bump k8s dependency in isolation. 
But I think for the code-generator in this case, it is probably fine as it mainly affect the cert-manager API related clients. I have tested it against k8s 1.14, and it works.
In addition, it introduces a minor change for Istio client (https://github.com/knative/serving/pull/6408/files#diff-9a2a4137b1328c89145c35e7743b2161R61), which looks fine. I can also run our E2E tests against k8s 1.14 and see if it works.
WDYT?

Reply:
We want the changes I mention this release anyways, I was waiting for 1.15 to land in GKE.  If it's there, then we should move now.  I believe we have issues in the 0.12.x milestone tracking the work.

Reply:
1.15 is already landed in GKE. I will start a separate PR to bump all k8s components to 1.16.